src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -41,67 +56,108 @@ export const ChannelClaimTableRow = ({ claimedChannel, channelType }: ChannelCla
     staleTime: Infinity,
   });
 
+  const channelName = isPublisherChannel ? publisherQuery.data?.name : serialPublicationQuery.data?.name;
+  const pendingChannelQuery = isPublisherChannel ? publisherQuery.isPending : serialPublicationQuery.isPending;
+
   const organizationQuery = useFetchOrganization(claimedChannel.claimedBy.organizationId);
   const organizationName = getLanguageString(organizationQuery.data?.labels);
 
   const publishingPolicy = claimedChannel.channelClaim.constraint.publishingPolicy;
   const editingPolicy = claimedChannel.channelClaim.constraint.editingPolicy;
 
-  const channelName = isPublisherChannel ? publisherQuery.data?.name : serialPublicationQuery.data?.name;
-
-  const pendingChannelQuery = isPublisherChannel ? publisherQuery.isPending : serialPublicationQuery.isPending;
+  const deleteMutation = useMutation({
+    mutationFn: async () => await deleteChannelClaim(customerIdentfier, channelIdentifier),
```

Comment:
Her kan det se ut til at det er lagt inn en del overflødig logikk? 🤔 

Vi vil sende en `DELETE <ChannelClaimId>` og vi har id fra før, i `claimedChannel.id`. I stedet for å bruke id på claimen vi har dras det ut identifier for både customer og channel, før de settes sammen til å bli id vi vil bruke. Er det en grunn til det? 🤔 

```suggestion
    mutationFn: async () => await deleteChannelClaim(claimedChannel.id),
```

Reply:
Det ligger ikke customerId i claimedChannel.id :/

Reply:
Så channel claim har en id men det er ikke den som skal brukes ved sletting? Høres snodig ut men mulig det er sånn for en grunn da. jeg kan høre litt :)